File: src/main/java/io/github/dqualizer/dqlang/types/adapter/ctk/SteadyStateHypothesis.java
package io.github.dqualizer.dqlang.types.adapter.ctk;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.ToString;

import java.util.List;
import java.util.Set;

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode
public class SteadyStateHypothesis {

    @JsonProperty("title")
    private String title;
    @JsonProperty("probes")
    private List<Probe> probes;
}

File: src/main/kotlin/io/github/dqualizer/dqlang/types/adapter/ctk/SteadyStateHypothesis.kt
package io.github.dqualizer.dqlang.types.adapter.ctk

import com.fasterxml.jackson.annotation.JsonProperty

data class SteadyStateHypothesis(
  @JsonProperty("title")
  val title: String? = null,

  @JsonProperty("probes")
  val probes: List<Probe>? = null
)
